src/Sentry/Platforms/Android/SentryOptions.cs

```diff
@@ -33,7 +33,7 @@ public class AndroidOptions
         /// <remarks>
         /// See https://docs.sentry.io/platforms/android/configuration/app-not-respond/
         /// </remarks>
-        public bool AnrReportInDebug { get; set; }
+        public bool AnrReportInDebug { get; set; } = false;
```

Comment:
why adding the redundant ` = false;`?

Reply:
To be explicit about matching the defaults that are set in the native SDKs.  Easy to miss an `= true` if there's a whole bunch of falses in a row. Better in this case to have a place for the default on every property.